index.js

```diff
@@ -151,8 +36,11 @@ app.post("/authorized_post_request", authMiddleWare, (req, res) => {
 });
 
 app.use("/", authRouter);
-
-// Listen for connections on specified port (default is port 4000)
+app.use('/client', clientRouter)
```

Comment:
just as an advice: if you're going to register a router to the `/` route, do it as the last one. If by any chance you have a route inside that router that is `/:id` for example, it would match basically everything and requests won't get to your routers below. by putting it last you make sure that first it didn't match any other more specific route